.github/workflows/flutter.yml

```diff
@@ -40,9 +40,11 @@ jobs:
         # https://github.com/flutter/flutter/issues/59522#issuecomment-646208247
         - name: Fix iOS build bug
           if: matrix.target == 'ios'
-          run: rm example/ios/Podfile
+          run: rm flutter/example/ios/Podfile
         - name: Test
-          run: flutter test
+          run: |
+            cd flutter
+            flutter test
```

Comment:
Line 52: `cd flutter/example`

Reply:
I thought that it'd remember cd flutter from line 46 :(